src/Subscriber/FilterShopwareExceptionIgnoresSubscriber.php

```
@@ -0,0 +1,43 @@
+    {
+    }
+
+    public static function getSubscribedEvents()
+    {
+        return [
+            FilterExceptionEvent::class => 'filterSentryExceptions'
+        ];
+    }
+
+
+    public function filterSentryExceptions(FilterExceptionEvent $event)
```

PHPStan (2.1.15)

Method Frosh\SentryBundle\Subscriber\FilterShopwareExceptionIgnoresSubscriber::filterSentryExceptions() has no return type specified.


(missingType.return)